backend/grant/task/jobs.py

```diff
@@ -0,0 +1,18 @@
+from grant.extensions import db
+
+
+def proposal_reminder(task_id):
+    from grant.task.models import Task
+    task = Task.query.filter_by(id=task_id).first()
+    from grant.proposal.models import Proposal
+    proposal = Proposal.query.filter_by(id=task.blob["proposal_id"]).first()
+    # TODO - replace with email
+    print(proposal)
+    task.completed = True
+    db.session.add(task)
+    db.session.commit()
+
+
+JOBS = {
+    1: proposal_reminder
+}
```

Comment:
Ideally these could be an enum, so when setting up a job, I pass `JOB_TYPE.PROPOSAL_REMINDER` instead of `1`.

Reply:
I haven't used enums in python a whole lot yet, but this does seems to be a good instance to use them.

However, wouldn't we still have to translate the enum value into an int to store it in the database? Or otherwise serialize it.

I'm not sure that there's a big enough value add to justify a translation step, or maybe there's another way to implement this.

Reply:
Enums can resolve to a value, e.g.
```
class MyEnum(Enum):
  proposal_reminder = 1
```